### PR TITLE
Pin alan version

### PR DESCRIPTION
**Pull request by aguillenv**
Fixes: https://github.com/alantech/meta/issues/164

### CODE REVIEW

**Review by dfellis — APPROVED** [review]

**Review by depombo — APPROVED** [review]
Amazing!

File: cli/src/main.rs [hunk 1]
```diff
@@ -62,7 +66,7 @@ pub async fn main() {
   let anycloud_agz = base64::encode(include_bytes!("../alan/anycloud.agz"));
   let app = App::new(crate_name!())
     .version(crate_version!())
-    .about("AnyCloud is a Lambda alternative that works with multiple cloud provider.")
+    .about(concat!("Elastically scale webservers in any cloud provider.\nCurrent alan version: ", alan_version!()))
```
Comment (depombo) [inline cli/src/main.rs:69]: nit: we can access the about cargo description as an environment variable so we only track it in one place https://stackoverflow.com/questions/27840394/how-can-a-rust-program-access-metadata-from-its-cargo-package

File: cli/src/main.rs [hunk 2]
```diff
@@ -62,7 +66,7 @@ pub async fn main() {
   let anycloud_agz = base64::encode(include_bytes!("../alan/anycloud.agz"));
   let app = App::new(crate_name!())
     .version(crate_version!())
-    .about("AnyCloud is a Lambda alternative that works with multiple cloud provider.")
+    .about(concat!("alan ", alan_version!(), "\n", env!("CARGO_PKG_DESCRIPTION")))
```
Comment (depombo) [inline cli/src/main.rs:69]: nit can we keep the format of your original message, so `concat!(env!("CARGO_PKG_DESCRIPTION"), ".\nCurrent alan version: ", alan_version!())`

Comment (aguillenv) [reply cli/src/main.rs]: ok!

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -13,6 +13,11 @@ jobs:
     outputs:
       upload_url: ${{ steps.create_release.outputs.upload_url }}
     steps:
+      - name: Get latest alan version
+        id: get_latest_alan
+        run: |
+          echo "::set-output name=alan_version::$(curl --silent "https://api.github.com/repos/alantech/alan/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')"
```
Comment (depombo) [inline .github/workflows/release.yml:19]: are we grabbing the entire string with the `v` included?

**Review by cdmistman — APPROVED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (aguillenv) [reply .github/workflows/release.yml:19]: Yes, even tho in this case it is only informational.

image

Comment (depombo) [reply .github/workflows/release.yml:19]: perfect!

**Review by depombo — APPROVED** [review]

**Pull request merged**